.github/workflows/lint.yml

```diff
@@ -0,0 +1,25 @@
+name: lint
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+
+jobs:
+  lint:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8
+    - name: Lint with flake8
+      run: |
+        flake8 . --count --max-line-length=99 --statistics
```

Comment:
I think the default line length limit of 79 characters is (1) undesirable in that it frequently results in contorted code formatting and (2) unnecessary now that it's 2022 and we have plenty of screen real estate at our disposal.  PEP8 has a brief discussion of this and allows `max-line-length` up to 99: https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Happy to change back to the default if you prefer though!

Reply:
Suits me just fine.

Reply:
They are removed!